packages/cli/src/commands/deployment/deploy.ts

```diff
@@ -48,8 +48,8 @@ export default class Deploy extends Command {
       description: 'The number of items kept in the cache before flushing',
       required: false,
     }),
-    indexerStoreCacheAsync: Flags.boolean({
-      description: 'If enabled the store cache will flush data asyncronously relative to indexing data.',
+    disableIndexerStoreCacheAsync: Flags.boolean({
+      description: 'If enabled the store cache will flush data asynchronously relative to indexing data.',
```

Comment:
I think this description comes from yargs, can you check for the same typo there